themes/bootstrap3/templates/RecordDriver/EDS/result-list.phtml

```diff
@@ -34,7 +35,30 @@
                   <h2>
                     <a href="<?=$this->escapeHtmlAttr($recordLinker->getUrl($this->driver))?>" class="title getFull _record_link"  data-view="<?=$this->escapeHtmlAttr($this->params->getOptions()->getListViewOption())?>">
                     <?=$item['Data']?> </a>
-                  </h2>
+		  </h2>
```

Comment:
I'm seeing a mix of tabs and spaces for indentation for a few lines starting here.  I think spaces are the standard for VuFind, but @demiankatz shouldn't php-cs-fixer or one of the other checks be catching this in the automated tests?

Reply:
@maccabeelevine, I think what's happening is that we're relying on `phpcbf` rather than `php-cs-fixer` for indentation normalization, but we don't apply `phpcbf` to templates. It looks like we've [explicitly disabled the statement_identation php-cs-fixer rule for templates](https://github.com/vufind-org/vufind/blob/dev/tests/vufind_templates.php-cs-fixer.php#L63) because it was misbehaving and garbling the files. This may be causing us to miss out on some issues. I'm not sure if anything in [this StackOverflow thread](https://stackoverflow.com/questions/42849484/use-tab-for-indentation-in-php-cs-fixer) might be used to improve the whitespace selection without breaking the overall file indentation; it might be worth experimenting with. I don't have time at this very moment, but if nobody else is interested in digging in deeper, let me know and I can put it on my to-do list for later.

Reply:
Thanks @demiankatz.  I followed that thread's advice (and some others) but no luck -- I can get it to change n!=2 spaces to 2, but not to change tabs :)  At least not in HTML.  Not any priority.

Reply:
Seems very possible that php-cs-fixer intentionally ignores anything outside of `<?php` tags, in which case we would need a different tool to solve HTML formatting issues in our templates.

Reply:
Thanks @demiankatz and @maccabeelevine , I've pushed fixes for those two points in https://github.com/vufind-org/vufind/pull/3841#discussion_r1713666675 . If you could please re-review now?